plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/submitit_launcher.py

```diff
@@ -99,16 +99,12 @@ def launch(
         queue_parameters = self.queue_parameters.copy()
         OmegaConf.set_struct(queue_parameters, True)
         if self.queue == "auto":
-            max_num_timeout = self.queue_parameters.auto.max_num_timeout
-            with open_dict(queue_parameters):
```

Comment:
This raised an error when trying on slurm.

Reply:
Thanks.
We need some proper testing for this plugin.